crates/aptos-logger/src/aptos_logger.rs

```diff
@@ -350,7 +356,13 @@ impl AptosDataBuilder {
             })
         };
 
-        crate::logger::set_global_logger(logger.clone());
+        let console_port = if cfg!(feature = "tokio-console") {
```

Comment:
I like that it's disabled by default 😄 The only remaining question in my mind is why do we need a compilation flag to enable it instead of a config value? If we're always going to compile it in to the binary, let's gate the feature using a config value (not cfg). But, if we want to avoid the compilation entirely, let's gate the whole thing at compile time?